data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java

```diff
@@ -100,18 +88,8 @@ public AuditRequestReceiver auditRequestReceiver(final AuditService auditService
     }
 
     @Bean
-    public DataReader dataReader(final CacheService cacheService) {
-        try {
-            return new HadoopDataReader().cacheService(cacheService);
-        } catch (IOException ex) {
-            LOGGER.error("Failed to instantiate HadoopDataReader: {}", ex.getMessage());
-            return new DataReader() {
-                @Override
-                public DataReaderResponse read(final DataReaderRequest dataReaderRequest, final Class<? extends Service> aClass, final AuditRequestCompleteReceiver auditRequestCompleteReceiver) throws NoCapacityException {
-                    return null;
-                }
-            };
-        }
+    public DataReader dataReader() throws IOException {
+        return new HadoopDataReader();
```

Comment:
Has the error handling here been removed intentionally and therefore is taken care of elsewhere?